packages/fxa-customs-server/lib/config/config.js

```diff
@@ -43,11 +43,25 @@ module.exports = function(fs, path, url, convict) {
     },
     limits: {
       blockIntervalSeconds: {
-        doc: 'Duration of a manual ban',
+        doc: 'Duration of a manual `block` ban',
         default: 60 * 60 * 24,
         format: 'nat',
         env: 'BLOCK_INTERVAL_SECONDS',
       },
+      suspectInterval: {
```

Comment:
In two minds about this suggestion. I can see the reasoning, but `blockIntervalSeconds` is a funny case where it is used by both the dataflow stuff and the non-dataflow, so it would seem wrong to nest it with the other two but also kind of wrong if they're nested together without it. So I'm leaning towards keeping them at this level, if that's cool.

Reply:
Thats cool with me, I didn't realize that the `blockIntervalSeconds` was used in both.